saltysimulations/salty-747
# fix(fmc): Inhibit VNAV when there is no CRZ ALT + ETA in POS Report fix

**Changes:**
- VNAV will not be inhibited when there is no CRZ ALT entered in the PERF INIT page.
- Fix ETA in Pos Report to not show UNDEFINED anymore. (Temporary fix until POS REPORT gets reworked)
**Testing instructions:**
- Load up aircraft, do not enter a CRZ ALT, check if VNAV does not activate.
- Check POS REPORT page, see if ETA shows UTC time instead of UNDEFINED.
